frontend/integration-tests/tests/event.scenario.ts

```diff
@@ -3,7 +3,7 @@ import { appHost, checkLogs, checkErrors, testName } from '../protractor.conf';
 import { execSync } from 'child_process';
 
 describe('Events', () => {
-  const name = `${testName}-pod`;
+  const name = `${testName}-event-test-pod`;
```

Comment:
Does this mess up the `afterAll()` func to clean up our pods?  Looks like its expecting a particular name.  
Next Q, does it matter....

Reply:
The `afterAll()` function specifically deletes the `name` value for the `namespace`, so it should work fine. 
```
execSync(`kubectl delete pods ${name} -n ${testName}`)
```

Reply:
👍